node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/InfrastructureProvisioner.java

```diff
@@ -23,17 +23,22 @@
     InfrastructureProvisioner(NodeRepository nodeRepository, InfraDeployer infraDeployer, Duration interval) {
         super(nodeRepository, interval);
         this.infraDeployer = infraDeployer;
+
+        // If this fails, we fail the component graph construction and bootstrap of config server,
+        // which is what we want.
+        try {
+            infraDeployer.activateAllSupportedInfraApplications(true);
```

Comment:
Consider moving this to `NodeRepositoryMaintenance` or remove https://github.com/vespa-engine/vespa/blob/6c122d41cb1d890c285ffd1786eb7cc16f8eba55/node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeRepositoryMaintenance.java#L90

Reply:
Nice catch. I renamed and kept the call from NodeRepositoryMaintenance().